### PR TITLE
Current page

### LINKED ISSUE #39: Current Bills Page -  /current
| Route    | `'/current'`|
| -------- | -------- |
| signed in only? | `true`|

Screenshot from 2020-06-24 15-10-27

#### Components:
 - Tips #32 
 - Bar Chart #36  
 - Bills Table #34

### PR DESCRIPTION
**Pull request by AhmedSafi97**
- added bar chart component to current page
- used statistical functions to 
  - distribute bills into groups and provide it the chart
  - calculate the trimmed mean
  - decide whether the user is in the heightest 10% consumption
  - give colors to the chart bars depending on each group frequency

 
closes #39

### CODE REVIEW

File: client/src/components/Charts/BarChart.js [hunk 1]
```diff
@@ -1,63 +1,69 @@
...
-            },
-          },
-        },
-        scales: {
-          xAxes: [
+const BarChart = ({ centers, frequencies, colors }) => {
+  const { t } = useTranslation();
+  return (
+    <div className="p-4 m-auto h-half-screen lg:w-2/3">
+      <Bar
```
Comment (MohammedIrheem) [inline client/src/components/Charts/BarChart.js:10]: split data and options like this 
<Bar data={data} options={options} />

Comment (MohammedIrheem) [inline client/src/components/Charts/BarChart.js:10]: split data and options like this 
<Bar data={data} options={options} />

Comment (MohammedIrheem) [inline client/src/components/Charts/BarChart.js:10]: **split data and options like this:**  
`<Bar data={data} options={options} />`

File: client/src/pages/Current.js [hunk 2]
```diff
@@ -1,45 +1,114 @@
...
+        const { data } = await axios.get(
+          `/api/v1/bills/${userId}/stats?typeId=${billId}&&billingMonth=${currentMonth}&&billingYear=${currentYear}`
+        );
+
+        const currentBillAmount = bills.filter(
+          (bill) =>
+            bill.billing_month === currentMonth &&
+            bill.billing_year === currentYear &&
+            bill.type_id === Number(billId)
+        )?.[0]?.amount;
```
Comment (Amoodaa) [inline client/src/pages/Current.js:40]: destructure

Comment (Amoodaa) [inline client/src/pages/Current.js:40]: i dont think we need them, if there is no user you shouldnt go down in the list

**Pull request merged**